docs/content/ChoiceGroup.mdx

```diff
@@ -0,0 +1,382 @@
+  </FormControl>
+  <FormControl>
+    <Radio name="radioInput" value="choiceTwo" />
+    <FormControl.Label>Choice two</FormControl.Label>
+  </FormControl>
+  <FormControl>
+    <Radio name="radioInput" value="choiceThree" />
+    <FormControl.Label>Choice three</FormControl.Label>
+  </FormControl>
+</ChoiceGroup>
+```
+
+### Using onChange handlers
+
+```javascript live noinline
+const WithOnChangeHandlers = () => {
+  const [selectedCheckboxValues, setSelectedCheckboxValues] = React.useState([])
+  const [selectedRadioValue, setSelectedRadioValue] = React.useState('')
+
+  const handleCheckboxChange = e => {
+    const {value, checked} = e.currentTarget
+
+    if (checked) {
+      setSelectedCheckboxValues([...selectedCheckboxValues, value])
+      return
+    }
+
+    setSelectedCheckboxValues(selectedCheckboxValues.filter(selectedValue => selectedValue !== value))
+  }
+
+  const handleRadioChange = e => {
+    const {value, checked} = e.currentTarget
+
+    if (checked) {
+      setSelectedRadioValue(value)
+    }
+  }
+
+  return (
+    <Box display="grid" sx={{gap: 3}}>
+      <Box display="grid" sx={{gap: 1}}>
+        <ChoiceGroup>
+          <ChoiceGroup.Label>Checkboxes</ChoiceGroup.Label>
+          <FormControl>
+            <Checkbox value="checkOne" onChange={handleCheckboxChange} />
+            <FormControl.Label>Checkbox one</FormControl.Label>
+          </FormControl>
+          <FormControl>
+            <Checkbox value="checkTwo" onChange={handleCheckboxChange} />
+            <FormControl.Label>Checkbox two</FormControl.Label>
+          </FormControl>
+          <FormControl>
+            <Checkbox value="checkThree" onChange={handleCheckboxChange} />
+            <FormControl.Label>Checkbox three</FormControl.Label>
+          </FormControl>
+        </ChoiceGroup>
+
+        {Boolean(selectedCheckboxValues.length) && (
+          <div>The selected checkbox values are {selectedCheckboxValues.join(', ')}</div>
+        )}
+      </Box>
+
+      <Box display="grid" sx={{gap: 1}}>
+        <ChoiceGroup>
+          <ChoiceGroup.Label>Radios</ChoiceGroup.Label>
+          <FormControl>
+            <Radio name="radioChoices" value="radioOne" onChange={handleRadioChange} />
+            <FormControl.Label>Radio one</FormControl.Label>
+          </FormControl>
+          <FormControl>
+            <Radio name="radioChoices" value="radioTwo" onChange={handleRadioChange} />
+            <FormControl.Label>Radio two</FormControl.Label>
+          </FormControl>
+          <FormControl>
+            <Radio name="radioChoices" value="radioThree" onChange={handleRadioChange} />
+            <FormControl.Label>Radio three</FormControl.Label>
+          </FormControl>
+        </ChoiceGroup>
+
+        {selectedRadioValue && <div>The selected radio value is {selectedRadioValue}</div>}
+      </Box>
+    </Box>
+  )
+}
+
+render(<WithOnChangeHandlers />)
+```
+
+### Checkbox group
```

Comment:
Should this one be merged into the basic example to appear alongside it? It feels like a 'basic' example to but quite tucked away in the examples.

Reply:
Yea, good suggestion.